nats-exporter/version.txt

```diff
@@ -1 +1 @@
-0.9.1
+0.9.1-1
```

Comment:
With this change should this be a 0.9.2?

Reply:
we are matching the version to nats exporter release the current one is the latest release
reference: https://github.com/nats-io/prometheus-nats-exporter/releases/tag/v0.9.1